WordPress/wordpress-playground
# Centralize log storage

## What is this PR doing?

It moves away from using the JS logger (console) to the `@php-wasm/logger`.

## What problem is it solving?

It centralizes log collections which will allow us to better utilize logs in the future.

## How is the problem addressed?

By replacing console calls with logger calls. 

## Testing Instructions

- Checkout this branch
- [Open this blueprint](http://localhost:5400/website-server/?blueprint-url=https://gist.githubusercontent.com/bgrgicak/5fbeb98abec8221192a8b619f852148a/raw/75ff00581b76d9ed3e4844d228dce9b4eda1b163/error-log-blueprint.json)
- Open the report error modal (menu in upper right corner > Report error)
- See JS logs in the _LOGS_ textarea

**Please review the log messages in this PR and suggest changes (using the GitHub suggest feature) to messages if they could be improved.**
